web/db/actions/Post.ts

```diff
@@ -83,23 +90,46 @@ async function getResizedUploadUrl(uuid: string): Promise<string> {
 }
 
 async function deleteAttachments(keysToDelete: string[]) {
-  //will make key splitting more efficient later
-  const objectsToDelete = keysToDelete.map((keyToDelete) => ({
-    Key:
-      keyToDelete.split("/")[keyToDelete.split("/").length - 2] +
-      "/" +
-      keyToDelete.split("/")[keyToDelete.split("/").length - 1],
-  }));
-  const deleteObjectsCommand = new DeleteObjectsCommand({
-    Bucket: consts.storageBucket,
-    Delete: { Objects: objectsToDelete },
-  });
-  try {
-    const returned = await storageClient.send(deleteObjectsCommand);
-    return { success: true };
-  } catch (error) {
-    return { success: false };
+  for (let i = 0; i < 2; i++) {
```

Comment:
Create a variable for max retries and use a while loop for readability

Reply:
The retry logic should only cover what's in the try catch right now - you won't need to remap the `objectsToDelete` and `deleteObjectsCommand`